metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorTableService.java

```diff
@@ -108,6 +128,116 @@ public void create(@Nonnull final ConnectorContext requestContext, @Nonnull fina
         }
     }
 
+    void updateTable(@Nonnull final ConnectorContext requestContext,
+                             @Nonnull final Table table,
+                             @Nonnull final TableInfo tableInfo) throws MetaException {
+        if (table.getParameters() == null || table.getParameters().isEmpty()) {
+            table.setParameters(Maps.newHashMap());
+        }
+        table.getParameters().putIfAbsent(PARAMETER_EXTERNAL, "TRUE");
+        if (tableInfo.getMetadata() != null) {
+            table.getParameters().putAll(tableInfo.getMetadata());
+        }
+
+        //storage
+        final StorageDescriptor sd = table.getSd() != null ? table.getSd() : new StorageDescriptor();
+        String inputFormat = null;
+        String outputFormat = null;
+        Map<String, String> sdParameters = Maps.newHashMap();
+        final String location =
+                tableInfo.getSerde() == null ? null : tableInfo.getSerde().getUri();
+        if (location != null) {
+            sd.setLocation(location);
+        } else if (sd.getLocation() == null) {
+            final String locationStr = hiveConnectorDatabaseService.get(requestContext,
```

Comment:
I do understand we spoke about this. Cant we use the hive client to get the database?